src/app/authoring-tool/edit-question-bank/edit-question-bank.component.html

```diff
@@ -9,6 +9,12 @@
     </mat-checkbox>
   </div>
   <div *ngIf="componentContent.questionBank?.enabled">
+    <div class="max-questions-to-show-div">
+      <mat-form-field class="form-field-no-hint" appearance="fill">
+        <mat-label i18n>Number of Questions to Show</mat-label>
+        <input matInput [(ngModel)]="componentContent.questionBank.maxQuestionsToShow" (ngModelChange)='saveChanges()' />
```

Comment:
Add a line break to this line because it's more than 100 columns wide.

Reply:
I think it would make sense to make an empty maxQuestionsToShow have it show all the questions. The maxQuestionsToShow is an optional added restriction. I think it would be ok to leave it blank by default and let the author specify but it may depend on how often authors choose to use that field and what they usually set it to.